waveform-django/waveforms/views.py

```diff
@@ -761,28 +759,28 @@ def render_annotations(request):
     ]
     all_anns_frac = f'{len(completed_annotations)}/{total_anns}'
     finished_assignment = len(completed_annotations) == total_anns
-
     if request.method == 'POST':
         if 'new_assignment' in request.POST:
+            record_dir = Path('./record-files/')
             available_projects = [p for p in all_projects if p not in base.BLACKLIST]
             num_events = int(request.POST['num_events'])
             assigned_events = {}
             unassigned_events = {}
 
             for project in available_projects:
                 assigned_events[project] = get_all_assignments(project)
+                project_dir = record_dir / project
 
-                records_path = os.path.join(PROJECT_PATH, project, base.RECORDS_FILE)
+                records_path = project_dir / base.RECORDS_FILE
```

Comment:
Had to read up on the library syntax to make sure this is platform independent :)